networkx/algorithms/centrality/tests/test_closeness_centrality.py

```diff
@@ -1,9 +1,10 @@
 """
-Tests for degree centrality.
+Tests for closeness centrality.
 """
 from nose.tools import *
 import networkx as nx
-
+import timeit
+from random import sample
```

Comment:
Tests with random parts are difficult to debug. Since these are unit-tests, it is OK to restrict it to a specific sequence of random values. So, use a seed with any call to a function with a random seed argument (like ```fast_gnp_random_graph```). Also, instead of using random.sample it is sufficient to use ```nx.utils.arbitrary_element(G)``` and then no random number is needed in that section.
The tests should not include timing so please remove that code (and the corresponding print statements).
You could leave it in the code as comments to show people how to time it if that is complicated, but in this case it doesn't seem too complicated to time the two approaches.

Thanks...